### DISCUSSION TITLE
Clarification about using JavaScript

### DISCUSSION THREAD

**Discussion by ajtruckle** [MicrosoftEdge/WebView2Feedback#4184] [Q&A] [answered]
I want to create a HTML file that will have some JavaScript. I am not sure of the code for the JavaScript yet. The plan is to have a paragraph displayed to the user and  based on a passed parameter on the URL use the JavaScript to load the correct localised text. I assume I can do this in Win32.

But, my concern is, is the setting of running JavaScript inside WebView2 separate from that of Edge or the system? Ie: can the user externally do something that will cause the script I don't want to run it in WebView2 of my app and it fail?

**Comment by gsgghotra** [comment, marked as answer]
First of all you can always add JavaScript with in the **<script> </script>** tags. And to display text on the page using JavaScript can be done via DOM manipulation. you can target the element and add text to it based on the input. And apologies for not answering the last question. I mean if the browser supports JavaScript, why would the app fail?

**Reply by ajtruckle** [reply]
@gsgghotra I tried to reword the last question. It can be ignored. 👍
